dotnet-client-libraries/YamlDotNet/App/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using App.Models;
+using App.UseCases;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NodeDeserializers;
+using static System.Console;
+
+namespace App;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var yamlProduct = BasicSerializeAndDeserialize.SerializeProduct(new Product
+        {
+            Id = 1,
+            Name = "Apple",
+            Price = 1.99m
+        });
+        WriteLine(yamlProduct);
+        var deserializeProduct = BasicSerializeAndDeserialize.DeserializeProduct(yamlProduct);
+        WriteLine($"Name: {deserializeProduct.Name}, Price: {deserializeProduct.Price}");
+
+        // Serialize and Deserialize Store Complex Object
+
+        var yamlStore = ObjectGraphSerializeAndDeserialize.SerializeStore(new Store
+        {
+            Name = "Tech Store",
+            Items =
+            [
+                new Item
+                {
+                    Name = "Laptop",
+                    Price = 1000m,
+                    Manufacturer = new Manufacturer
+                    {
+                        Name = "Tech Corp",
+                        Country = "USA"
+                    }
+                },
+
+                new Item
+                {
+                    Name = "Smartphone",
+                    Price = 500m,
+                    Manufacturer = new Manufacturer
+                    {
+                        Name = "Mobile Inc",
+                        Country = "China"
+                    }
+                }
+            ]
+        });
+
+        WriteLine(yamlStore);
+
+        var deserializeStore = ObjectGraphSerializeAndDeserialize.DeserializeStore(yamlStore);
+
+        WriteLine($"Store: {deserializeStore.Name}");
+
+        foreach (var item in deserializeStore.Items)
+        {
+            WriteLine($"Item: {item.Name}, Price: {item.Price}");
+            WriteLine($"Manufacturer: {item.Manufacturer.Name}, Country: {item.Manufacturer.Country}");
+        }
+
```

Comment:
Could you remove the blank line here, and a few lines down, so that there's not 2 blank lines in a row?

Reply:
@grantwinney Done